File: cleiton031/programacao_basica/questao17.py
print("Cofrinho do Pedrinho")

u = float(input("Informe a qtd de moedas de 1 centavo:"))
c = float(input("Informe a qtd de moedas de 5 centavos:"))
d = float(input("Informe a qtd de moedas de 10 centavos:"))
v = float(input("Informe a qtd de moedas de 25 centavos:"))
ci = float(input("Informe a qtd de moedas de 50 centavos:"))
ur = float(input("Informe a qtd de moedas de 1 real:"))

u2 = u * 0.01
c2 = c * 0.05
d2 = d * 0.10
v2 = v * 0.20
ci2 = ci * 0.50
ur2 = ur * 1.00

r = u2 + c2 + d2 + v2 + ci2 + ur2

print( )
print("O resultado total de dinheiro eh:" + str(r))

File: felipe029/lista_fluxograma/Questoes_java/Questao17.java
import java.util.Scanner;

public class Questao17 {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);

        System.out.print("Informe a qtd de moedas de 1 centavo:");
        double u = s.nextInt();

        System.out.print("Informe a qtd de moedas de 1 real: ");
        double ur = s.nextInt();

        System.out.print("Informe a qtd de moedas de 5 centavo: ");
        double c = s.nextInt();

        System.out.print("Informe a qtd de moedas de 10 centavo: ");
        double d = s.nextInt();

        System.out.print("Informe a qtd de moedas de 25 centavo: ");
        double v = s.nextInt();

        System.out.print("Informe a qtd de moedas de 50 centavo: ");
        double ci = s.nextInt();

        double u2 = u * 0.01;
        double c2 = c * 0.05;
        double d2 = d * 0.10;
        double v2 = v * 0.20;
        double ci2 = ci * 0.50;
        double ur2 = ur * 1.00;

        double r = u2 + c2 + d2 + v2 + ci2 + ur2;

        System.out.println("\nO resultado total de dinheiro eh:" + r);
    }
}
